examples/hotrod-containerized/README.md

```diff
@@ -0,0 +1,41 @@
+# Hot R.O.D. - Rides on Demand - Containerized Version
```

Comment:
Why new directory `hotrod-containerized`? 

it can live in `examples/hotrod` and just add instructions to the original readme

Reply:
+1, I'd prefer all in one directory

Reply:
OK. Changing to one hotrod directory